tests/foreman/ui/test_repository.py

```diff
@@ -96,6 +96,7 @@ def test_create_repo_basic(self):
                         org=self.organization.name,
                         name=repo_name,
                         product=prod.name,
+                        product_instance=self.products,
```

Comment:
I have two points here:

1. If repositories will have some dependency with products, what about passing it to repositories constructor?
2. What about in repositories constructor, repository creates a reference to repositories using the same browser that it have?

I still thinking on a better solution here, what I have for now is the felling that passing product_instance every call id not something good to require.